packages/xsnap/test/test-xs-perf.js

```diff
@@ -55,18 +55,10 @@ test('meter details', async t => {
     {
       compute: 'number',
       allocate: 'number',
-      allocateChunksCalls: 'number',
-      allocateSlotsCalls: 'number',
-      garbageCollectionCount: 'number',
-      mapSetAddCount: 'number',
-      mapSetRemoveCount: 'number',
-      maxBucketSize: 'number',
     },
     'auxiliary (non-consensus) meters are available',
```

Comment:
nit: this `'auxiliary (non-consensus) meters are available'` test description is no longer quite accurate